src/hepattn/models/decoder.py

```diff
@@ -202,16 +221,16 @@ def __init__(
         attn_norm = norm if not hybrid_norm else None
         dense_post_norm = not hybrid_norm
 
-        attn_kwargs = attn_kwargs or {}
+        self.attn_kwargs = attn_kwargs or {}
```

Comment:
I'd rather not store the `kwargs` as a member, let's just pull out what we need and store that.